cmd/commands/subvolume.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	subvolume "github.com/rook/kubectl-rook-ceph/pkg/filesystem"
+	"github.com/spf13/cobra"
+)
+
+var SubvolumeCmd = &cobra.Command{
+	Use:   "subvolume",
+	Short: "manages the stale-subvolumes",
+	Args:  cobra.ExactArgs(1),
+}
+
+var listCmd = &cobra.Command{
+	Use:   "ls",
+	Short: "Print the list of stale subvolumes no longer in use.",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := cmd.Context()
+		clientsets := GetClientsets(ctx)
+		VerifyOperatorPodIsRunning(ctx, clientsets, OperatorNamespace, CephClusterNamespace)
+		staleSubvol := cmd.Flag("stale").Value.String()
```

Comment:
maybe use this, so that the conversion to a `bool` is done early on

```
staleSubvol, err := cmd.Flag("stale").GetBool()
```

Reply:
I tried it but, saw some error, I dont think we have GetBool()